crates/sui-core/src/consensus_adapter.rs

```diff
@@ -292,21 +293,19 @@ impl ConsensusAdapter {
                 .tap_err(|r| {
                     error!("Submit transaction failed with: {:?}", r);
                 })?;
-
+            self.num_inflight_transactions
+                .fetch_add(1, Ordering::SeqCst);
             // Increment the attempted certificate sequencing
             self.opt_metrics.as_ref().map(|metrics| {
                 metrics.sequencing_certificate_attempt.inc();
+                metrics
+                    .sequencing_certificate_inflight
+                    .set(self.num_inflight_transactions.load(Ordering::Relaxed) as i64);
```

Comment:
Nit: you can reuse value returned by fetch_add above to avoid potentially more expensive loading